releases/2022.2/2022.2.md

```diff
@@ -0,0 +1,177 @@
+---
+title: 2022.2
+parent: Releases
+---
+
+<div align="center">
+  <h1>Kytos-ng 2022.2 - kiko</h1>
+</div>
+
+<div align="center">
+  <a href="https://www.amlight.net"><img src="https://amlight.net/wp-content/uploads/2016/02/AmLlightExP_s.jpg"></a>
+</div>
+
+Kytos-ng[^1] version `2022.2`, kiko, was released in August.
+
+## Objectives
+
+This version had these major objectives:
+
+- Optimize `mef_eline` EVCs convergence to take place in a few seconds on average.
+- Integrate a production-grade recommended document-oriented database for Napps.
+- Enhance Kytos core to support event message prioritization and have higher performance when handling events in general.
+- Have a better infrastructure tools for application performance monitoring (APM) and logging.
+- New major features such as link liveness detection, interface loop detection, and new Web UI components.
+- Increase unit test coverage to at least 85% on `coloring`, `sdntrace`, and `sdntrace_cp`.
```

Comment:
Wouldn't it be on all Napps ? (then, to achieve the goal we had to work on coloring, sdntrace, and sdntrace_cp). I mean: now all Kytos Napps have more than 85% coverage, right? Maybe we can rephrase this to give this idea.

Reply:
That's a good way to put it, @italovalcy. `flow_stats` is an exception though if we phrase it as all Kytos NApps, since we strategically decided not to try to cover it since it'll have major refactoring in the future. Maybe we can claim that in the next release, or maybe we can also come up with another adjective here, like all non-experimental NApps, and then we tag `flow_stats` with an `experimental` stability badge, let me know what you prefer to have here. 

I was also going to suggest to say all stable NApps, but some NApps haven't got the `stable` badge yet.

Reply:
Very interesting point, @vinicius. What do you think about leaving it as "all Kytos Napps (except flow_stats, which has a major refactoring coming soon)"?

Reply:
Great idea @italovalcy, let's go for it.

Reply:
@italovalcy done here https://github.com/kytos-ng/kytos-ng.github.io/pull/34/commits/4980a9a785cd3fab3ec99da0dd3f1a7d0597acbb

Reply:
@italovalcy done here https://github.com/kytos-ng/kytos-ng.github.io/pull/34/commits/4980a9a785cd3fab3ec99da0dd3f1a7d0597acbb

Reply:
I'll mark this as resolved. Thanks.